spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -620,10 +680,13 @@ def get_options(self):
                     show_all_files=self.treewidget.show_all_files,
                     group_cells=self.treewidget.group_cells,
                     show_comments=self.treewidget.show_comments,
+                    sort_files_alphabetically=\
+                        self.treewidget.sort_files_alphabetically,
                     expanded_state=self.treewidget.get_expanded_state(),
-                    scrollbar_position=self.treewidget.get_scrollbar_position(),
+                    scrollbar_position=\

```

Comment:
PEP 8 and I believe our modern conventions these days strongly recommends never using ``\`` and always using ``()`` instead, and that's what's generally used in recently edited files.